python/paddle/tensor/linalg.py

```diff
@@ -3350,3 +3350,116 @@ def corrcoef(x, rowvar=True, name=None):
         c = paddle.clip(c, -1, 1)
 
     return c
+
+
+def cdist(
+    x, y, p=2.0, compute_mode="use_mm_for_euclid_dist_if_necessary", name=None
+):
+    r"""
+
+    Compute the p-norm distance between each pair of the two collections of inputs.
+
+    This function is equivalent to `scipy.spatial.distance.cdist(input,'minkowski', p=p)`
+        if :math:`p \in (0, \infty)`. When :math:`p = 0` it is equivalent to `scipy.spatial.distance.cdist(input, 'hamming') * M`.
+        When :math:`p = \infty`, the closest scipy function is `scipy.spatial.distance.cdist(xn, lambda x, y: np.abs(x - y).max())`.
+
+    Args:
+        x (Tensor): Tensor, shape is :math:`B \times P \times M`.
+        y (Tensor): Tensor, shape is :math:`B \times R \times M`.
+        p (float, optional): The value for the p-norm distance to calculate between each vector pair. Default: :math:`2.0`.
+        compute_mode (str, optional): the mode for compute distance.
+            - 'use_mm_for_euclid_dist_if_necessary', for p = 2.0 and (P > 25 or R > 25), it will use
+                matrix multiplication to calculate euclid distance if possible.
+            - 'use_mm_for_euclid_dist', for p = 2.0, it will use matrix multiplication to calculate euclid distance.
+            - 'donot_use_mm_for_euclid_dist', it will not use matrix multiplication to calculate euclid distance.
+            Default: 'use_mm_for_euclid_dist_if_necessary'.
```

Comment:
同样格式不对导致解析错误
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/b60c92e1-27fc-46c2-9a68-de8a6d78178d)
同时，建议参数可选值置灰
```suggestion
        compute_mode (str, optional): the mode for compute distance.
            
            - ``use_mm_for_euclid_dist_if_necessary`` , for p = 2.0 and (P > 25 or R > 25), it will use
            matrix multiplication to calculate euclid distance if possible.
            - ``use_mm_for_euclid_dist', for p = 2.0`` , it will use matrix multiplication to calculate euclid distance.
            - ``donot_use_mm_for_euclid_dist`` , it will not use matrix multiplication to calculate euclid distance.
            
            Default: ``use_mm_for_euclid_dist_if_necessary``.
```